HW02/Alexey_Zalyotov/alexey_zalyotov_hw_02_t_03.rb

```diff
@@ -0,0 +1,34 @@
+require 'date'
+
+def events_timestamps
```

Comment:
We run this method many times in the code. Have you considered using caching??
```suggestion
def events_timestamps
  @events_timestamps ||= ...
```